app/src/main/java/com/hamdan/forzenbook/viewmodels/LoginViewModel.kt

```diff
@@ -96,33 +98,53 @@ class LoginViewModel @Inject constructor(
     }
 
     fun submitLogin() {
+//        viewModelScope.launch {
```

Comment:
Don't leave commented blocks in, especially without a TODO with associated ticket

Reply:
fair, left a ToDo